### PR TITLE
Todo app

### PR DESCRIPTION
**Pull request by ChrispaixK**
# To do App
> A simple todo app in javascript wich allow users to add tasks edit them or mark them as completed

In this PR I : 
* Setup the working environment
* Add functionalities of the app to 
  * Add a task
  * Edit a task
  * Mark the task as complete
* Style the full app
* Add responsiveness and theme switcher
* Test the app

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+    return tasks;
+  }
+
+  // add the task to store
+  static addTask(task) {
+    const tasks = Store.getTasks(); // get tasks
+    tasks.push(task); // add new task
+    localStorage.setItem('tasks', JSON.stringify(tasks)); // convert them in string
+    // document.querySelector('.tasks-left-num').textContent = tasks.length;
```
Comment (benshidanny11) [inline src/store.js]: [Required] Kindly remove the unwanted comment in order to keep the code clean.

File: src/store.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      }
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    for (let i = 0; i < tasks.length; i += 1) {
+      if (tasks[i].index >= spliced[0].index) {
+        tasks[i].index -= 1;
+      }
+    }
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    // document.querySelector('.tasks-left-num').textContent = tasks.length;
```
Comment (benshidanny11) [inline src/store.js:39]: [Required] The same here.  The unwanted comments should be removed in order to keep the code clean!

File: src/store.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+// Store Class: Handles Storage
+class Store {
+  // Get Tasks : gets the tasks from store
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      // if there is nothing in storage
+      tasks = []; // create empty array
+    } else {
```
Comment (benshidanny11) [inline src/store.js]: **[Required]** Here you are writing a lot of lines of code to perform one operation. You can use a single line to achieve this 
Ex: 
`const tasks= localStorage.getItem('tasks') || [];`

And then return the tasks. Kindly let's fix it 👌

**Review by benshidanny11 — CHANGES_REQUESTED** [review]
Hi @ChrispaixK  ✋ , 

_Great job so far 👏 💯 👍. However, there are a few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - The app functions well. 💯


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
  - [ ] Putting all js files in one folder to keep the project well structured
  - [ ] Centering the App title in the UI in order to make the app looks great.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/store.js (hunk 1 above)
Comment (ChrispaixK) [reply src/store.js]: thank you for the feedback @benshidanny11

File: src/store.js (hunk 2 above)
Comment (ChrispaixK) [reply src/store.js:39]: thank you for the feedback @benshidanny11  , I actually used them to keep an eye on what I was doing, sorry for that

File: src/store.js (hunk 3 above)
Comment (ChrispaixK) [reply src/store.js]: thank you for the feedback @benshidanny11

**Review by benshidanny11 — APPROVED** [review]
Hi @ChrispaixK ,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Highlights
  - Follows git workflow.  💯
  - The codebase looks clean and professional. 💯 
  - The App functions well. 💯 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**Pull request merged**